content/zotero_intro_to.md

```diff
@@ -384,7 +384,7 @@ We will take **10 minutes** for this exercise.
 
 ### Going further
 
-**|!| Tip : let Zotero works for you**
+**💡 Tip : let Zotero works for you**
```

Comment:
to be left textual :point_right: find an Atom package supporting emoji when converting to PDF

Reply:
Ok